Library/Formula/opencoarrays.rb

```diff
@@ -0,0 +1,73 @@
+class Opencoarrays < Formula
+  desc "Coarray Fortran runtime library and toolchain, free & open source"
+  homepage "http://opencoarrays.org"
+  url "https://github.com/sourceryinstitute/opencoarrays/releases/download/1.2.0/opencoarrays-1.2.0.tar.gz"
+  sha256 "7d0c3f2dd842b32b2a24ac5fb923bc026d541d9e037205162b39a3d381d9fa83"
+
+  head "https://github.com/sourceryinstitute/opencoarrays.git"
+
+  option "without-test", "Run build time tests (requires properly configured MPICH and firewall settings)"
+
+  depends_on :fortran
+  depends_on "gcc"
```

Comment:
There is no need to explicitly depend on `gcc`. As far as I can tell, the build system only uses the MPI wrappers anyway. Also, `depends_on :fortran` in practice already implies a `gcc` dependency.